java/BUILD

```diff
@@ -125,6 +134,15 @@ alias(
     actual = ":java11_debug_debian9",
 )
 
+[[container_test(
+    name = "java_base" + mode + distro_suffix + "_test",
+    configs = ["testdata/java_base_debug.yaml" if ("debug" in mode) else "testdata/java_base.yaml"],
```

Comment:
Perhaps you can just do `"testdata/java_base" + mode + ".yaml"`?

Reply:
Done 👍